src/web_app_skeleton/src/actions/errors/show.cr.ecr

```diff
@@ -104,27 +99,11 @@ class Errors::Show < Lucky::ErrorAction
   end
 
   # This is the catch all method that renders unhandled exceptions
-  def handle_error(error : Exception) : Lucky::Response
-    <%- if api_only? -%>
+  def render(error : Exception) : Lucky::Response
     render_unhandled_error(error)
-    <%- else -%>
-    if Lucky::ErrorHandler.settings.show_debug_output && html?
```

Comment:
This is now handled in Lucky::ErrorAction. If html and  show_debug_output it will *always* show  debug output. Disable debug output to see what users will see